app/scripts/modules/kubernetes/container/configurer.directive.html

```diff
@@ -32,6 +32,46 @@
     </div>
   </div>
 
+  <collapsible-section heading="Basic Settings" expanded="false">
+    <div class="form-group">
+      <div class="sm-label-left">
+        <b>Environment Variables</b>
+      </div>
+      <table class="table table-condensed">
+        <thead>
+        <tr>
+          <th>Name</th>
+          <th>Value</th>
+          <th></th>
+        </tr>
+        </thead>
+        <tbody>
+        <tr ng-repeat="pair in container.envVars">
```

Comment:
Any reason you can't reuse the `<map-editor>` here?

Here are a couple of spots where it is being used:

https://github.com/spinnaker/deck/blob/master/app/scripts/modules/google/serverGroup/configure/wizard/advancedSettings/advancedSettings.directive.html#L35-L41
https://github.com/spinnaker/deck/blob/master/app/scripts/modules/amazon/serverGroup/configure/wizard/advancedSettings/advancedSettings.html#L134-L140